### ISSUE TITLE
Porovnání úřadů a organizací (ne z hlediska K-indexu, ale vůbec)

### ISSUE THREAD

**Issue by lenka-crypto** [HlidacStatu/Hlidac-Statu#99]
Nefunguje, ale fungovalo 
(via Eliščin manuál, který jsem zatím skryla, než se to opraví - https://texty.hlidacstatu.cz/wp-admin/post.php?post=5056&action=edit)

**Comment by suchoss** [comment]
Jo, o tom víme, ale to porovnání (jestli myslíme to stejné, protože článek jsi schovala dobře a já se na něj nedostanu) moc dobře nefungovalo a nešlo přenést do .NET core. Musíme vymyslet smysluplnější. Nápady, co by to mělo umět klidně pište sem :)
